elasticgraph-warehouse/lib/elastic_graph/warehouse/schema_definition/field_type_converter.rb

```diff
@@ -19,13 +19,20 @@ class FieldTypeConverter
         #
         # @param field_type [Object] the field type to convert
```

Comment:
```suggestion
        # @param field_type [ElasticGraph::SchemaDefinition::SchemaElements::TypeReference] the field type to convert
```

I realized `Object` wasn't correct here.  Only `TypeReference` has an `unwrap_non_null` method (which is immediately called on `field_type`).